local-test.js

```diff
@@ -35,3 +36,5 @@ if (!firestoreData) {
   }
   await docRef.delete()
 })()
+
+logger.info('SUCCESS!!! backend setup successful')
```

Comment:
Are you sure this is on the right line? Have you tried it out?
Shouldn't it be inside the async function?

Reply:
Yeah right, it should be inside async function. Required changes done.